packages/tables/frontend/sirius-components-tables/src/table/useTableColumns.tsx

```diff
@@ -27,7 +27,8 @@ export const useTableColumns = (
       return {
         id: column.id,
         header: column.label,
-        size: 150,
+        size: column.width ?? column.initialWidth,
```

Comment:
why can't that be done deep in the backend instead of propagating everywhere a potential null value? And why two values?